tests/test_rest_server.py

```diff
@@ -654,3 +654,431 @@ def client(role='admin', timeout=0.1):
     for res_file in results:
         total_size += res_file["catalog"]["file_size"]
     assert total_size < 1000000000
+
+@pytest.mark.asyncio
+async def test_bundles_bulk_crud(mongo, rest):
+    """Check CRUD semantics for bundles."""
+    mongo.Bundles.delete_many(ALL_DOCUMENTS)
```

Comment:
is there a better way to make this automated?  maybe put this in the fixture, so it's not in every test?

Reply:
I'm unclear as to what we want to refactor here?
The data sets used by testing functions?
Or make some omni-CRUD routine where we provide the type?

Reply:
the line `mongo.Bundles.delete_many(ALL_DOCUMENTS)`, which gets repeated at the start of every test.

Reply:
would be nice to assume that the DB is empty at the start of a test (and for the mongo fixture to "make it so")

Reply:
Ahhh, OK. That should be easy enough.

Reply:
Done: https://github.com/WIPACrepo/lta/pull/25/commits/76fc3e039b8ace59f905d1b1e328cd1cccc6129f